src/options/components/navigation/index.jsx

```diff
@@ -19,44 +34,94 @@ const Navigation = ({ currentLocation, routes }) => {
             })
 
             return (
-                <li className={navClasses} key={idx}>
-                    <i className={navIcon}>{route.icon}</i>
-                    {route.component === 'faq' && (
-                        <a
-                            className={navClasses}
-                            href={route.pathname}
-                            target="_blank"
-                        >
-                            {route.name}
-                        </a>
-                    )}
-                    {route.component !== 'faq' && (
-                        <Link className={navClasses} to={route.pathname}>
-                            {route.name}
-                        </Link>
-                    )}
+                <li>
```

Comment:
All the JSX code in this container should move to their own separate views. Ideally in React you should separate views from unrelated container logic, making the views simple functions of whatever props get passed in. State derivation and event handlers should be taken care of away from the concerns of the view. Really simple state-less view example is [`ResultsMessage.js`](https://github.com/WorldBrain/WebMemex/blob/master/src/overview/components/ResultsMessage.js).

Here I can see maybe 3 different views you could split the JSX into (names are just suggestions):
- `<Nav>` like the overall container view
- `<NavLink>` a view just representing one of the nav items/links (many would appear inside of `<Nav>`)
- `<NavLinkSection>` a view just representing one of the nested link items for imports (many could conditionally appear inside `<NavLink>`)

Easy way to think about refactoring this: As soon as you can delete the `import styles from './styles.css'` line, you have moved all the views away :)